use/quickstart.md

```diff
@@ -5,28 +5,171 @@ permalink: /use/quickstart/
 redirect_from: /getting-started/
 ---
 
-# Apache Beam Quickstart
+# Apache Beam Java SDK Quickstart 
 
-The Apache Beam project is in the process of bootstrapping. This includes the creation of project resources, the refactoring of the initial code submission, and the formulation of project documentation, planning, and design documents. Until the project is fully initialized, this page contains useful resources to learn more about the model and tools which comprise Apache Beam.
+Run your first Beam pipeline to count words in a set of files. 
```

Comment:
A little more intro text?

This Quickstart will walk you through executing your first Beam pipeline, written using Beam's Java SDK, on a runner of your choice. For the purposes of this Quickstart, we'll use the sample [WordCount pipeline](link), which counts words in a set of text files, but these instructions apply to other pipelines as well.